### DISCUSSION TITLE
Why this in iframe dev?

### DISCUSSION THREAD

**Discussion by xunziheng** [wxt-dev/wxt#747] [Q&A] [answered]
seems like wxt framework inserted those scripts.
<img width="650" alt="image" src="https://github.com/wxt-dev/wxt/assets/35691153/7537563d-4edb-497b-8ee1-be15f55aa99b">

**Comment by aklinker1** [comment, marked as answer]
Fix released in v0.18.7, see https://github.com/wxt-dev/wxt/pull/723. This was a bug in the new modules API, plugins are runtime code injected by modules.
